src/index.js

```diff
@@ -6,6 +6,9 @@ const fs = require('fs');
 const feedQueue = require('./feed-queue');
 const feedWorker = require('./feed-worker');
 
+// Start the web server
+require('./backend/web/server');
```

Comment:
We are not starting the server here. This isn't needed.

Reply:
For this PR they do need to start it.  The Queues and Express need to exist in the same context, so that Bull Queues are in scope.  I told them to do this here, just to get it working.  We'll refactor next week when we get all this code landed and integrate.

Reply:
sounds good to me